teraserver/python/modules/FlaskModule/API/service/ServiceQueryParticipants.py

```diff
@@ -66,6 +67,11 @@ def get(self):
             if participant:
                 return participant.to_json()
 
+        if args['participant_id']:
+            participant = TeraParticipant.get_participant_by_id(args['participant_id'])
+            if participant:
+                return participant.to_json()
+
```

Comment:
This should be removed - when possible, service API should use UUIDs instead of IDs.